doc/clusters/redis.md

```diff
@@ -79,6 +79,10 @@
 ## Cluster create flow
 To create a Redis cluster instance you need to prepare the yaml manifest. Here is an example:
 
+Notice:
+- If name is not specified in spec, the name of cluster will be the same as metadata name.
+- If you decided to omit the name, make sure that metadata name is much the pattern on Instaclustr.
```

Comment:
Maybe it will be better replace it with following text, wdyt? 
 "If you choose to omit the cluster name from the specification, the operator will use it from the metadata name instead. In this case make sure that metadata name matches the name pattern, you can check the pattern on instaclustr API specification"

Reply:
Maybe it will be better replace it with following text, wdyt? 
 "If you choose to omit the cluster name from the specification, the operator will use it from the metadata name instead. In this case make sure that metadata name matches the name pattern, you can check the pattern on instaclustr API specification"